packages/cli/src/utils/skills/skills-offer.ts

```
@@ -0,0 +1,238 @@
+    write: (message) => proxy.write(message),
+    // Closes the underlying interface directly: the proxy throws once the
+    // stream is closed, and close must stay callable from the finally block.
+    close: () => rl.close(),
+  }
+}
+
+function getCliVersion(): string {
+  try {
+    // eval keeps esbuild from inlining the require; the path resolves
+    // relative to the built CLI bundle, next to which package.json lives.
+    return eval(`require('../package.json')`).version
```

Biome (2.5.1)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)